stacker_blueprints/dynamodb.py

```diff
@@ -0,0 +1,152 @@
+from stacker.blueprints.base import Blueprint
+from . import util
+
+from troposphere import (
+    dynamodb2,
+    Ref,
+    GetAtt,
+    Output,
+)
+
+from troposphere.route53 import RecordSetType
+
+def prep_schemata(config):
+    try:
+        schemata = []
+        for schema in config["KeySchema"]:
+            schemata.append(dynamodb2.KeySchema(**schema))
+
+    except IndexError:
+        raise IndexError(
+            "The key schema is required for the creation of a DynamoDB table."
```

Comment:
would use `KeySchema` instead of `key schema` since you're defining a specific required value.

I wonder if we should use the new concept of `validators` to handle this validation and return the troposphere type, with all of this prep work done at that point